app/views/reset_pins/change_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>Change PIN</h1>
+<%= form_tag change_pin_path, method: :post do %>
+  <%= hidden_field_tag :token, params['token'] %>
+  <div>
+    Your new PIN must:
+    <ul>
+      <li>be 6-25 characters long</li>
```

Comment:
These rules aren't actually validated anywhere, but its more of a current feature parity thing. We could add future validation if we need to.